move_base/include/move_base/move_base.h

```diff
@@ -76,6 +76,61 @@ namespace move_base {
     OSCILLATION_R
   };
 
+  //Used to calculate distance, time and average velocity
+  struct FeedbackInfo
```

Comment:
2-spaces tabulation needed in all this struct.

Reply:
Thanks